app/src/main/res/values-night/colors.xml

```diff
@@ -80,7 +80,7 @@
     <color name="top_site_title_text">@color/top_site_title_text_dark_theme</color>
     <color name="top_site_pager_dot">#3A3944</color>
     <color name="top_site_pager_dot_selected">#5B5B66</color>
-    <color name="top_site_pin_icon_background_tint">@color/photonViolet50</color>
+    <color name="top_site_pin_icon_background_tint">@color/foundation_light_theme</color>
```

Comment:
I think we should rename this to be 
```suggestion
    <color name="top_site_pin_icon_color">@color/foundation_light_theme</color>
```
since it's no longer used in the `android:backgroundTint` property